PurchasesTests/Purchasing/PurchasesTests.swift

```diff
@@ -928,28 +928,28 @@ class PurchasesTests: XCTestCase {
         setupPurchases()
         let product = MockSKProduct(mockProductIdentifier: "com.product.id1")
 
-        var purchaserInfo: PurchaserInfo?
+        var purchaserInfo: CustomerInfo?
         var receivedError: Error?
         var receivedUserCancelled: Bool?
 
-        let purchaserInfoBeforePurchase = PurchaserInfo(data: [
+        let purchaserInfoBeforePurchase = CustomerInfo(data: [
             "request_date": "2019-08-16T10:30:42Z",
             "subscriber": [
                 "first_seen": "2019-07-17T00:05:54Z",
                 "original_app_user_id": "app_user_id",
                 "subscriptions": [:],
                 "non_subscriptions": [:]
             ]])
-        let purchaserInfoAfterPurchase = PurchaserInfo(data: [
+        let purchaserInfoAfterPurchase = CustomerInfo(data: [
```

Comment:
we should rename these variables as well

Reply:
💯 
✅